robot/basestation/templates/elements/controls/ManualControls.html

```diff
@@ -1,120 +1,140 @@
 <!-- Manual controls section -->
 <table class="manual-ctrl-grp" style="display: none;" id="ArmcontrolsON" align="center">
-    <tr>
-      <!-- motor 3 -->
-        <td>
-            <a href="javascript:void(0)" id="click_btn_motor3_ccw">
-                <button type="button" class="rect-btn control-btn-up fa fa-chevron-up fa-3x"></button>
-            </a>
-        </td>
-        <td>
-            <button type="button" class="control-btn-mid" disabled>
-                <strong>M3 Elbow up/down</strong>
-            </button>
-        </td>
-        <td>
-            <a href="javascript:void(0)" id="click_btn_motor3_cw">
-                <button type="button" class="rect-btn control-btn-down fa fa-chevron-down fa-3x"></button>
-            </a>
-        </td>
 
-        <td style="width:30px"></td>
-
-        <!-- motor 4 -->
-        <td>
-            <a href="javascript:void(0)" id="click_btn_motor4_ccw">
-                <button type="button" class="rect-btn control-btn-up fa fa-chevron-up fa-3x"></button>
-            </a>
-        </td>
-        <td>
-            <button type="button" class="control-btn-mid" disabled>
-                <strong>M4 Wrist up/down</strong>
-            </button>
-        </td>
-        <td>
-            <a href="javascript:void(0)" id="click_btn_motor4_cw">
-                <button type="button" class="rect-btn control-btn-down fa fa-chevron-down fa-3x"></button>
-            </a>
-        </td>
-    </tr>
+    <!-- First row: Motors names -->
     <tr>
-      <!-- motor 2 -->
+
       <td>
-          <a href="javascript:void(0)" id="click_btn_motor2_ccw">
-              <button type="button" class="rect-btn control-btn-up fa fa-chevron-up fa-3x"></button>
-          </a>
+          <button type="button" class="control-btn-mid" disabled>
```

Comment:
control-btn-_mid_ makes it seem that it is something related to the _middle_ of something. which doesnt appear to be the case